repologyapp/templates/docs/bots.html

```diff
@@ -59,7 +59,7 @@ <h4>repology-linkchecker</h4>
 <a href="https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods">HEAD</a>
 request first, and only if that fails it falls back to a
 <a href="https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods">GET</a>
-request. This means that it most cases the robot won't retrieve the contents of an URL,
```

Comment:
I though _an_ comes before a vowel, what am I missing here? (same question for _an usable_)

Reply:
@AMDmi3 https://www.quora.com/Why-dont-we-use-an-before-a-hard-u-sound-like-in-Universe